integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,158 @@
+# Run Automation Test on Kobiton devices with Travis CI
+This document will guide you how to use Travis CI to execute automation test on Kobiton devices everytime you push a commit to GitHub.
+
+## Table of contents 
+  - [A. Preparation](#a-preparation)
+    - [1. Prerequisites](#1-prerequisites)
+    - [2. Sample](#2-sample)
+  - [B. Setup](#b-setup)
+    - [1. Setting environment variables](#1-setting-environment-variables)
+    - [2. Getting desired capabilities](#2-getting-desired-capabilities)
+    - [3. Configuring automation test project](#3-configuring-automation-test-project)
+  - [C. Execution](#c-execution)
+    - [1. Run automation script on Kobiton devices](#1-run-automation-script-on-kobiton-devices)
+    - [2. Fetch session data through REST API](#2-fetch-session-data-through-rest-api)
+  - [D. Feedback](#d-feedback)
+
+# A. Preparation
+
+## 1. Prerequisites
+
+  - Kobiton username and API key.
+  
+    Please visit https://portal.kobiton.com/register to create a new Kobiton account if you do not have one.
+
+    Follow instructions at `IV. Configure Test Scripts for Kobiton` section on [our blog article](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/) to get Kobiton username and API key. They are necessary for authenticating with Kobiton devices.
+
+  - A GitHub repository for automation testing that is fully configured with Travis CI.
+  
+    If you don't have one, please follow the instruction on the [next section](#2-sample) to get a sample.
+
+## 2. Sample
+We have provided a sample for automation test with Travis CI configuration file in this repository. If you already got one, please skip this step.
+
+Note that this script is written in Node.js for testing a sample Android app. Please make your own adjustment if you are using other languages and apps.
+
+To use the provided sample, follow these steps:
+
+1. Fork this repository https://github.com/kobiton/Travis-HockeyApp-Appium-ReactNative/tree/master
+2. Synchronize the forked repository with Travis CI. Visit [Travis CI document](https://docs.travis-ci.com/user/legacy-services-to-github-apps-migration-guide/#how-to-migrate-a-private-repository-to-github-apps) for detailed instruction.
+3. Follow the next sections to get `desiredCaps` variable and replace to your script for running automation test.
+
+# B. Setup
+
+  **Purposes**
```

Comment:
If you what to push a purpose to explain the session, try to create a  flow that help user can understand the reason of each part. With the current flow, it hard to understand